providers/ns1/ns1Provider.go

```diff
@@ -158,6 +211,21 @@ func (n *nsone) modify(recs models.Records, domain string) error {
 	return err
 }
 
+// Configures DNSSEC for a zone. Set 'enabled' to true to enable, false to disable.
+// There's a cornercase, in which DNSSEC is globally disabled for the account.
+// In that situation, enabling DNSSEC will always fail with:
+//   #1: ENABLE DNSSEC
+//   FAILURE! POST https://api.nsone.net/v1/zones/example.com: 400 DNSSEC support is not enabled for this account. Please contact support@ns1.com to enable it
+//
+// Unfortunately this is not detectable otherwise, so given that we have a nice error message, we just let this through.
+//
+func (n *nsone) configureDNSSEC(domain string, enabled bool) error {
+	z, _, err := n.Zones.Get(domain)
```

Comment:
Check `err`.  (otherwise, it is overwritten by line 225)

Reply:
good catch!